README.md

```diff
@@ -88,7 +88,9 @@ docker run --rm -it -v \
 ## Usage
 
 ### Input data 
-In order to train or evaluate a model you have to provide data. 
+In order to train a model, evaluate a model or predict the quality you have to 
```

Comment:
"predict the quality of its output"

Reply:
The quality of an MLC result is predicted. Therefore it is not the output of the model itself, but the output of an MLC model.
Perhaps rephrase to: "predict the quality of an MLC result"

Reply:
okay -- as long as "the quality of what" is specified